splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -140,14 +143,14 @@ private void configureHandler() throws Exception {
                         .authenticationPreemptive(true)
                         .authMethod("Basic")
                         .httpMethod("POST")
-                        .authUsername("x")
+                        .authUsername("unused")
```

Comment:
is this still needed if you are setting authorization above?

Reply:
Good point. No there is not need to keep this!